packages/engine/src/scene/components/GroundPlaneComponent.ts

```diff
@@ -43,16 +44,15 @@ export const GroundPlaneComponent = defineComponent({
 
   onInit(entity) {
     return {
-      color: new Color(),
+      color: 0xffffff as ColorRepresentation,
```

Comment:
with this change, make sure the studio still works correctly, as the editor might be expecting a color object

Reply:
Ya, the editor color input has been updated to work with a color representation